Userland/Libraries/LibWeb/CSS/StyleValue.h

```diff
@@ -97,6 +97,47 @@ struct EdgeRect {
     Gfx::FloatRect resolved(Layout::Node const&, Gfx::FloatRect) const;
 };
 
+namespace Filter {
+struct Blur {
```

Comment:
Insert some newlines between namespaces and structs, they are free :smile:

Reply:
I charge per LoC, including whitespace :stuck_out_tongue_closed_eyes: